mcache/datadog_checks/mcache/mcache.py

```diff
@@ -105,10 +104,6 @@ class Memcache(AgentCheck):
 
     SERVICE_CHECK = 'memcache.can_connect'
 
-    @classmethod
-    def get_library_versions(cls):
-        return {"memcache": pkg_resources.get_distribution("python-binary-memcached").version}
```

Comment:
This function is entirely removed, is there no alternative to using `pkg_resources` to implement this function?

Reply:
Each entry in `importlib.metadata.distributions()` iterator should have `version` attribute. See also [version](https://docs.python.org/3/library/importlib.metadata.html#distribution-versions) function.

Reply:
this was not being used by anything so I simply removed it